src/helpers/randomSlug.ts

```diff
@@ -0,0 +1,40 @@
+import axios from 'axios';
+
+import {
+  getFromSessionStorage,
+  removeFromSessionStorage,
+  saveToSessionStorage,
+} from './sessionStorage';
+
+type RandomPostQueryResult = {
+  slug: string;
+  excludedSlugs: string[];
+};
+
+const RANDOM_POSTS_KEY = 'randomPosts';
+const RANDOM_POSTS_SLUG_PATH = '/api/posts/random';
+
+const randomSlug = async (): Promise<string> => {
```

Comment:
I think helper should be a simple thing.
For this one is coming with making a request, I think it should turn into another thing such as a service.
What do you think?